mws/mws.py

```diff
@@ -279,6 +291,68 @@ def enumerate_param(self, param, values):
                 params['%s%d' % (param, (num + 1))] = value
         return params
 
+    def enumerate_dict(self, param, dic):
+        """
+        Builds a dictionary of an enumerated parameter. Takes any dictionary and returns a dictionary recursively
```

Comment:
Add example like you did in enumerate_list. It took me some time to understand the purpose of the function.